addons/calendar/models/calendar_event.py

```diff
@@ -590,6 +590,13 @@ def action_mass_mailing(self):
             'context': compose_ctx,
         }
 
+    def action_join_meeting(self):
+        """ Method used when an existing user wants to join
+        """
+        partner = self.env.user.partner_id
```

Comment:
I would add an self.ensure_one(), even if it would work on a multi, seems more tailored but a singleton.